src/backend/env/mod.rs

```diff
@@ -194,7 +197,6 @@ pub struct State {
 
     migrations: BTreeSet<UserId>,
 
-    #[serde(default)]
```

Comment:
Why we don't need this anymore?

Reply:
This annotation is only needed for new fields. Then their values are persisted and deserialized from the stable memory.